config/example/ir.yaml

```diff
@@ -19,6 +19,45 @@ morph:
   validators: # List of hex-encoded 33-byte public keys of sidechain validators to vote for at application startup
     - 0283120f4c8c1fc1d792af5063d2def9da5fddc90bc1384de7fcfdda33c3860170
   switch_interval: 2m # interval b/w RPC switch attempts if the node is not connected to the highest priority node
+  consensus: # Local consensus launch mode. Activates only when 'endpoint.client' is unset.
+    block_interval: 1s # Time period (approximate) between two adjacent blocks. Must be positive
+    magic: 15405 # Network magic. Must be unsigned integer in range [1:4294967295]
+    traceable_chain_length: 2102400 # Length of the chain accessible to smart contracts. Must be in range [1:4294967295]
+    committee: # Initial committee
+      - 02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2 # Hex-encoded public key
+    seed_nodes: # Optional list of existing nodes to communicate with over Neo P2P protocol. By default, node runs as standalone
+      # Same format as 'p2p.listen'
+      - node2:20333
+      - node3:20333
+    hardforks: # Optional hardforks
+      name: 1730000 # Maps name to chain height. Heights must not be greater than 4294967295
+    storage: # Blockchain storage
+      type: boltdb # One of following storage types:
+        # boltdb (local BoltDB)
+        # leveldb (local LevelDB)
+        # inmemory (volatile storage, mostly suitable for testing)
+        #
+        # If 'path' exists, it should correspond to the selected variant.
+      path: ./db/morph.bolt # File system path to the storage. Ignored for 'inmemory' storage.
+    rpc: # RPC settings
+      listen: # Optional list of network addresses to listen Neo RPC on. By default, protocol is not served
```

Comment:
Overall RPC is required by storage layer to work. Theoretically we can disable RPC in some (not all) nodes, but I don't know why. Correct me if I'm wrong @AnnaShaleva @roman-khimov.

Reply:
IR node can't work without RPC, so it's always enabled. Whether it's listening or not is a separate matter, in most of the cases it will, becase SNs need to connect via RPC to something as well.